content/docs/pulumi-cloud/get-started/what-is-it.md

```diff
@@ -0,0 +1,111 @@
+---
+title_tag: "Pulumi Cloud: What is it?"
+meta_desc: Learn how Pulumi Cloud relates to the open source Pulumi infrastructure as code tool.
+title: "What is it?"
+h1: "What is Pulumi Cloud, and how does it relate to Pulumi's open source IaC tool?"
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    cloud:
+        name: "What is it?"
+        parent: pulumi-cloud-get-started
+        weight: 1
+        identifier: pulumi-cloud-what-is-it
+---
+
+So you've chosen to use Pulumi infrastructure as code, and are now deciding how to manage your state and whether Pulumi Cloud is a good fit? Or you've heard Pulumi is open source but aren't entirely clear on what part is fully open and free, and what is a paid Pulumi product? This page will give you a better understanding about the answers to these questions and everything in between.
+
+## In Summary
+
+Pulumi Cloud helps teams adopt collaborative, secure, and robust cloud engineering practices. Pulumi's infrastructure as code tool is [open source](https://github.com/pulumi), and by default it leverages Pulumi Cloud to make adopting IaC in your team easier, secure, and reliable out-of-the-box.
```

Comment:
I don't think the definition of Pulumi Cloud in the first sentence is clear. I think we should answer with Pulumi Cloud is...

Reply:
Pulumi Cloud is a fully managed infrastructure management platform that covers three products...

probably some of the stuff in the 3rd paragraph can be moved into this paragraph